### PR TITLE
feat(grouping): implement drag-and-drop task board

### PR DESCRIPTION
**Pull request by ChunWarayut**
- Add Grouping Board component with column structure
- Create TypeScript interfaces for tasks and columns
- Set up basic styling and layout
- Implement drag-and-drop functionality

### CODE REVIEW

**Comment by ChunWarayut** [comment]
@permlap-phola MR KUB

**Comment by permlap-phola** [comment]
bug

- [ ] item start to reoder without move to to correct position
- [ ] implement clone shadow of element when element is being draged.

File: package.json [hunk 1]
```diff
@@ -11,19 +11,21 @@
...
-    "framer-motion": "^11.11.13",
     "copy-webpack-plugin": "^12.0.2",
+    "framer-motion": "^11.11.13",
     "next": "14.2.5",
     "next-nprogress-bar": "^2.3.13",
     "nookies": "^2.5.2",
     "primereact": "^10.8.3",
     "qrcode": "^1.5.4",
     "react": "^18",
+    "react-beautiful-dnd": "^13.1.1",
```
Comment (permlap-phola) [inline package.json:28]: delete unused package

File: package.json [hunk 2]
```diff
@@ -39,6 +41,7 @@
     "@types/node": "^20",
     "@types/qrcode": "^1.5.5",
     "@types/react": "^18",
+    "@types/react-beautiful-dnd": "^13.1.8",
```
Comment (permlap-phola) [inline package.json:44]: delete unused package

**Comment by ChunWarayut** [comment]
@permlap-phola fix ไปแล้วครับ

**Pull request merged**